deployment.yml

```diff
@@ -22,10 +22,10 @@ spec:
             memory: 20Mi
         env:
           - name: MAX_POD_DURATION
-            value: 4m
+            value: 2m
           - name: POLL_INTERVAL
-            value: 10s
+            value: 30ms
```

Comment:
Good catch! It was that low so I could test the race condition. But there is no reason for it to be that low normally (I think I even bumped up the default poll interval) will fix ASAP